src/docfx.website.themes/default/partials/head.tmpl.partial

```diff
@@ -14,6 +14,7 @@
   <link rel="stylesheet" href="{{_rel}}styles/main.css">
   <meta property="docfx:navrel" content="{{_navRel}}">
   <meta property="docfx:tocrel" content="{{_tocRel}}">
+  {{#_noindex}}<meta name="searchOption" content="noindex">{{/_noindex}}
```

Comment:
we won't support nofolloe

---
In reply to: [134405842](https://github.com/dotnet/docfx/pull/1983#discussion_r134405842) [](ancestors = 134405842)